account/user.go

```diff
@@ -165,3 +165,18 @@ var TestUser = User{
 	Email:    "testdeveloper@testalm.io",
 	Identity: TestIdentity,
 }
+
+// TestIdentity2 only creates in memory obj for testing purposes
+var TestIdentity2 = Identity{
```

Comment:
No pb. I am just trying to help where I can at this point. This has even more impact when you look at the Docs... as those variables are in the documentation :-/ https://godoc.org/github.com/almighty/almighty-core/account#pkg-variables